.github/workflows/ci.yml

```diff
@@ -0,0 +1,73 @@
+name: "build-test"
+on:
+  push:
+    branches:
+      - net203
+  pull_request:
+    branches:
+      - net203
+
+jobs:
+  test-backend:
+    runs-on: ${{ matrix.image }}
+    strategy:
+      matrix:
+        image: [windows-latest]
+    env:
+      NUGET_PACKAGES: ${{ github.workspace }}/.github/nuget-packages
+      GRADLE_USER_HOME: ${{ github.workspace }}/.github/gradle
+    steps:
+      - uses: actions/checkout@v2
+
+      # Prepare
+      - name: Gradle JVM Cache
+        uses: actions/cache@v1.1.0
+        with:
+          path: build/gradle-jvm
+          key: ${{ runner.os }}.gradle-jvm.${{ hashFiles('rider-fsharp/gradle*') }}
+
+      - name: Gradle Wrapper Cache
+        uses: actions/cache@v1.1.0
+        with:
+          path: ${{ env.GRADLE_USER_HOME }}/wrapper
+          key: ${{ runner.os }}.gradle-wrapper.${{ hashFiles('rider-fsharp/gradle/**') }}
+
+      - name: NuGet Cache
+        uses: actions/cache@v1.1.0
+        with:
+          path: ${{ env.NUGET_PACKAGES }}
+          key: ${{ runner.os }}.nuget.${{ hashFiles('ReSharper.FSharp/**/*.csproj') }}+${{ hashFiles('ReSharper.FSharp/**/*.props') }}
+
+      - name: Gradlew Prepare
+        working-directory: ./rider-fsharp
+        run: ./gradlew prepare
+
+      - name: Setup .NET Core SDK
+        uses: actions/setup-dotnet@v1.7.2
+        with:
+          dotnet-version: '3.1.101'
+
+      - name: Add MSBuild to PATH
+        uses: microsoft/setup-msbuild@v1.0.2
```

Comment:
Do we need to add MSBuild to the path? We don't use it for the plugin build itself.
Is it for the tests run? Could you add a comment if yes?

Reply:
yes this is for tests